d2l-button-icon.js

```diff
@@ -112,11 +112,26 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-button-icon">
 				opacity: 0.5;
 			}
```

Comment:
@svanherk, does this commit look OK for the patch file? Is there anything to add/remove?

Reply:
Yeah I think this commit is good.  If we have to rerun the script the `FIXME` comments will be added back, but that's fine, we've just been leaving them in the other repos.